packages/text-annotator/src/model/w3c/W3CTextFormatAdapter.ts

```diff
@@ -171,6 +173,6 @@ export const serializeW3CTextAnnotation = <E extends W3CTextAnnotation = W3CText
     created: created?.toISOString(),
     modified: updated?.toISOString(),
     target: w3cTargets
-  } as E;
+  } as unknown as E;
```

Comment:
The `as unknown` coercion is justified here to handle the `...rest` destruction above. It'll have the following broad type: 
<img width="566" alt="image" src="https://github.com/user-attachments/assets/16fd12fe-b342-4a7e-902c-f0ee3ee08c42">
So the TS won't be able to know that the `I` actually fulfills the `E` type constraint.